### ISSUE TITLE
Save perplexity on training and validation sets

### ISSUE THREAD

**Issue by philipcori** [lvapeab/nmt-keras#129]
Hello, I was wondering if it is possible to save the perplexity measure on both the validation set as well as the training set. As of now it only saves the values to the file 'val.perplexity', however I would also like to save the training perplexity, rather than just seeing the values during training in the logs. Setting ```params['EVAL_ON_SETS']  = ['val', 'train']``` doesn't seem to work. EVAL_ON_SETS_KERAS doesn't seem to save it either. Having this would be very useful to see if my model is under/over-fitting. Any help on this would be really appreciated!

**Comment by lvapeab** [comment]
If you set perplexity as a keras metric (`KERAS_METRICS = ['perplexity']`), it should be logged in Tensorboard (by default, in `path-to-model/tensorboard_logs`).

**Comment by philipcori** [comment]
Got it, thanks.

**Comment by philipcori** [comment]
Sorry, even though I have tensorboard working, I only see loss and perplexity on training data now in tensorboard. I set ```params['EVAL_ON_SETS_KERAS'] = ['val', 'train']```, however this doesn't seem to change anything.
![image](https://user-images.githubusercontent.com/12155278/79668684-72755f80-816b-11ea-9ed0-6eb436cb7f9c.png)

**Comment by lvapeab** [comment]
Try updating MKW (`pip install -U multimodal-keras-wrapper`) and setting `METRICS= ['perpleity']`.

The parameter `KERAS_METRICS` is used for logging of the training, while `METRICS` evaluates the model predictions (typically on the `val` set). The `EVAL_ON_SETS` parameters refers to `METRICS`. 

This is rather convoluted and confusing. Will try to clarify it in a future.

**Comment by philipcori** [comment]
Ok it works now after updating multi-modal-keras. Though I had to pip install directly from the repo, for some reason ```pip install -U multimodal-keras-wrapper``` did not successfully update.
